frontend/scripts/react-components/dashboard-element/dashboard-element.saga.js

```diff
@@ -85,7 +126,10 @@ export function* onItemChange(action) {
   if (panel === 'countries') {
     yield fork(getDashboardPanelSectionTabs, dashboardElement, panel);
   } else if (items && !activeItemExists) {
-    yield fork(getDashboardPanelData, dashboardElement, panel);
+    // only reload list if clearing item from a different panel than the one that is active
+    // if (!activeItem && panel !== dashboardElement.activePanelId) {
```

Comment:
I just realised this is commented, can't understand why this line is here. If we uncomment it leaving the if statement, it also works. I wonder if this is necessary? Thoughts @Bluesmile82 ?